x-pack/lib/config_management/elasticsearch_source.rb

```diff
@@ -13,8 +13,9 @@
 module LogStash
   module ConfigManagement
     class ElasticsearchSource < LogStash::Config::Source::Base
-      include LogStash::Util::Loggable, LogStash::LicenseChecker::Licensed,
-              LogStash::Helpers::ElasticsearchOptions
+      include LogStash::Util::Loggable, LogStash::LicenseChecker::Licensed, LogStash::Helpers::ElasticsearchOptions
```

Comment:
Nit: https://github.com/rubocop/ruby-style-guide#mixin-grouping

```suggestion
      include LogStash::Util::Loggable
      include LogStash::LicenseChecker::Licensed
      include LogStash::Helpers::ElasticsearchOptions
```